turbopack/crates/turbo-tasks-fs/src/lib.rs

```diff
@@ -474,7 +474,7 @@ impl DiskFileSystem {
     /// # Arguments
     ///
     /// * `name` - Name of the filesystem.
-    /// * `root` - Path to the given filesystem's root.
+    /// * `root` - Path to the given filesystem's root. Should be canonicalized.
```

Comment:
'canonicalized' meaning no `..`?, no symlinks? an absolute path?

Reply:
Added a link to `std::fs::canonicalize`: https://doc.rust-lang.org/nightly/std/fs/fn.canonicalize.html